src/commands/coffeechats/coffeeGenerate.ts

```diff
@@ -0,0 +1,35 @@
+import { Message } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { generateFutureMatches } from '../../components/coffeechat';
+import { AdminCommand } from '../../utils/commands';
+import { parseNumArg, validateNumArg } from './utils';
+
+class coffeeSignupCommand extends AdminCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'coffee-generate',
+      aliases: ['coffeegenerate'],
+      group: 'coffeechats',
+      memberName: 'generate',
+      description: 'Generates coffeechat matchings for the next N weeks',
```

Comment:
I thought we would try to generate as many weeks as possible before we run into a dupe?